GPU/GPUTracking/Global/GPUChainTrackingClusterizer.cxx

```diff
@@ -1073,4 +1172,4 @@ int32_t GPUChainTracking::RunTPCClusterizer(bool synchronizeOutput)
 
 #endif
   return 0;
-}
+}
```

Comment:
Your editor seems to regular misformat the files removing newlines at end of files. Could you please have a look and fix this?